src/main/java/org/sunbird/halloffame/service/HallOfFameServiceImpl.java

```diff
@@ -146,6 +146,63 @@ public SBApiResponse fetchingTop10Learners(String ministryOrgId, String authToke
         return response;
     }
 
+    @Override
+    public SBApiResponse getUserLeaderBoard(String orgId) {
+        SBApiResponse response = ProjectUtil.createDefaultResponse(Constants.API_HALL_OF_FAME_ORG_READ);
+        if (orgId == null || orgId.isEmpty()) {
+            setBadRequestResponse(response, Constants.INVALID_ORG_ID);
+            return response;
+        }
+        Map<String, Object> propertyMap = new HashMap<>();
+        propertyMap.put(Constants.ORGID, orgId);
+        propertyMap.put(Constants.DB_COLUMN_ROW_NUM, Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10));
```

Comment:
Why this is set this way explain?

Reply:
this is to fetch only first 10 values from the table (specifically first for row_num column).
this is to show rank for the leader board, where first 10 only will be considered.